qiskit/visualization/pulse/qcstyle.py

```diff
@@ -17,48 +17,62 @@
 """
 Style sheets for pulse visualization.
 """
-from typing import Tuple, List
+from collections import namedtuple
+from typing import Tuple, Union, Optional
+
+import logging
+
+logger = logging.getLogger(__name__)
+ComplexColors = namedtuple('ComplexColors', ['real', 'imaginary'])
+SchedTableColors = namedtuple('SchedTableColors', ['time', 'channel', 'event'])
 
 
 class SchedStyle:
     """Style sheet for Qiskit-Pulse schedule drawer."""
-    def __init__(self, figsize: Tuple[float, float] = (10, 12),
+    def __init__(self, figsize: Optional[Tuple[float, float]] = (10.0, 12.0),
                  fig_unit_h_table: float = 0.4,
                  use_table: bool = True,
                  table_columns: int = 2,
                  table_font_size: int = 10,
                  axis_font_size: int = 18,
                  label_font_size: int = 10,
                  icon_font_size: int = 18,
-                 title_font_size: int = 25,
+                 title_font_size: Optional[Union[int, str]] = 25,
```

Comment:
why this takes `str`? is this `Union[int, None]` ?
https://www.python.org/dev/peps/pep-0484/#using-none

Reply:
`Optional[T]` is equivalent to `Union[T, None]` see: https://docs.python.org/3/library/typing.html#typing.Optional for more information.  The reason that It's `Union[int, str]` is because the names, `large`, `small`, are allowed to be used.  See https://matplotlib.org/3.2.1/users/dflt_style_changes.html#figure-size-font-size-and-screen-dpi.  If you think it's confusing to use the rcParams, then I can just tell require the user to use ints instead.

Reply:
I see. I feel allowing `str` is confusing without this information. I think we can stick to `int` here because string representation is vague.